.github/workflows/run-tests.yaml

```
@@ -0,0 +1,95 @@
+# This workflow will run all tests as well as pre-commit 
+
+name: Tests and linting
+on:
+  [push, pull_request]
```

I just noticed that when using Travis there was a [cron job](https://travis-ci.org/github/mozilla/OpenWPM/builds) running every 2 days. If we want to keep doing this I think something like the following would work:

```suggestion
on:
  push:
  pull_request:
  schedule:
    - cron: '0 0 */2 * *'
```